components/button.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Icon link -->
+    <link rel="icon" href="../assets/icon.png">
+    <!-- css style links -->
+    <link rel="stylesheet" href="../css/base_styles/reset.css">
+    <link rel="stylesheet" href="../css/base_styles/home.css">
+    <link rel="stylesheet" href="../css/component_styles/button.css">
+    <link rel="stylesheet" href="../css/component_styles/alert.css">
+    <link rel="stylesheet" href="../css/base_styles/introduction.css">
+    <link rel="stylesheet" href="../css/base_styles/media.css">
+
+    <!-- title -->
+    <title>Button</title>
+
+    <!-- Font Awesome link -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+        integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V" crossorigin="anonymous" />
+</head>
+
+<body>
+    <!-- navbar-section -->
+    <nav class="navigation">
+        <div class="nav-brand"><a class="link" href="/index.html">Rebalance-UI</a></div>
+        <ul class="non-bullet">
+            <li class="list-tem-inline"><a class="link link-active" href="/index.html">Home</a></li>
+            <li class="list-tem-inline"><a class="link" href="./avatar.html">Docs</a></li>
+        </ul>
+    </nav>
+
+    <!-- side-list bar -->
+    <div class="aside-container">
+        <aside class="aside-nav">
```

Comment:
use `main` element for all your main content and nest `aside` inside `main` element.